tests/test_discount.py

```diff
@@ -0,0 +1,31 @@
+from pywr.parameters import Parameter, load_parameter
+import datetime
+
+class DiscountFactorParameter(Parameter):
+    """Parameter that returns the current discount factor based on discount rate and a base year.
+
+    Parameters
+    ----------
+    discount_rate : discount rate fixed across planning period
+    base_year : base year in 'yyyy' format
```

Comment:
I think your code wants this as an `int`.
```
base_year : int
    Discounting base year (i.e. the year with a discount factor equal to 1.0).
```

Reply:
Thanks, I have updated the description.